app/src/androidTest/java/ch/epfl/cs311/wanderwave/model/ProfileConnectionTest.kt

```diff
@@ -112,7 +109,8 @@ public class ProfileConnectionTest {
             "profilePictureUri" to "https://example.com/image.jpg",
             "topSongs" to listOf<DocumentReference>(),
             "chosenSongs" to listOf<DocumentReference>(),
-        )
+            "bannedSongs" to listOf<DocumentReference>(),
+            "likedSongs" to listOf<DocumentReference>())
```

Comment:
Also references likedSongs and fails test locally

Reply:
It passes for me, I don't know why, either way I deleted